Makefile

```diff
@@ -11,7 +11,7 @@ go.vet:
 	$(DQLITE_BUILD_SCRIPTS_DIR)/static-go-vet.sh ./...
 
 go.test:
-	$(DQLITE_BUILD_SCRIPTS_DIR)/static-go-test.sh -v -p 1 ./...
+	$(DQLITE_BUILD_SCRIPTS_DIR)/static-go-test.sh -v -p 1 ./... -coverprofile=coverage.txt --cover --coverpkg=./...
```

Comment:
Why do we need to change the default behavior of `make go.test`?

Reply:
I could absolutely make a separate Markefile target like `go.coverage` but having coverage generated in the normal `go.test` target has negligible performance impact so I didn't see the point.

Could absolutely separate it if you'd like.